epinar/assignment1/q2.py

```diff
@@ -0,0 +1,63 @@
+
+class Node:
+
+	def __init__(self, data):
+		self.data = data
+		self.next = None
+
+class LinkedList:
+
+	def __init__(self, head = None):
+		self.head = head
+		self.curr = head
+		self.size = 1 if head else 0
+
+	def add(self, data):
+		new_node = Node(data)
+		if self.size == 0 : 
+			self.head = new_node
+		else:
+			self.curr.next = new_node
+
+		self.curr = new_node
+		self.size += 1
+
+		return True
+
+	def print(self):
+		node = self.head
+		while node:
+			print(node.data)
+			node = node.next
+
+	def lastKth(self, k):
+
+		stop = self.head
+		get = self.head
+		for i in range(k):
+			if stop == None:
+				return None
+			stop = stop.next
+
+		while stop.next:
+			get = get.next
+			stop = stop.next
+
+		return get.data
+
+def main():
```

Comment:
Please create unit tests, I think the way you create the linked list, you revert the order of the input list.

Reply:
Unit tests created, not sure why you thought like that.